File: feature-demos/skill-connections/requester-demo/lambda/handlers/LaunchRequestHandler.js
/**
 * Handler to handle LaunchRequest.
 */
const LaunchRequestHandler = {
    canHandle(handlerInput) {
      return handlerInput.requestEnvelope.request.type === 'LaunchRequest';
    },
    handle(handlerInput) {
      console.log("LaunchRequest received.");
      const speechText = 'Welcome to Skill Connections Requester Skill.';
      return handlerInput.responseBuilder
        .speak(speechText)
        .getResponse();
    }
};

module.exports = LaunchRequestHandler;

File: feature-demos/reminders-api/skill-demo-recurring-reminders/src/com/amazon/ask/channelguide/handlers/LaunchRequestHandler.java
package com.amazon.ask.channelguide.handlers;

import com.amazon.ask.dispatcher.request.handler.HandlerInput;
import com.amazon.ask.dispatcher.request.handler.RequestHandler;
import com.amazon.ask.model.LaunchRequest;
import com.amazon.ask.model.Response;

import java.util.Optional;

import static com.amazon.ask.request.Predicates.requestType;

public class LaunchRequestHandler implements RequestHandler {
    @Override
    public boolean canHandle(HandlerInput input) {
        return input.matches(requestType(LaunchRequest.class));
    }

    @Override
    public Optional<Response> handle(HandlerInput input) {
        String speechText = "Welcome to Channel Guide. You can ask, when is the next episode of Kings and Thrones. Which show would you like to know more information about?";
        String repromptText = "Which show would you like to know more information about?";
        return input.getResponseBuilder()
                .withSimpleCard("Channel Guide", speechText)
                .withSpeech(speechText)
                .withReprompt(repromptText)
                .build();
    }
}
